src/framework/ui/view/keynavigationsubsection.cpp

```diff
@@ -132,6 +137,7 @@ void KeyNavigationSubSection::setSection(KeyNavigationSection* section)
     m_section = section;
 
     if (m_section) {
+        m_section->addSubSection(this);
```

Comment:
When setting the section property of a sub-section object, this sub-section object is added to the specified section.
Previously, I added a subsection to a section in the componentCompleted method, because before adding it I planned to check for the correctness of the state ( a name, order, unique order, etc.).
But now the section of the sub-section can be set dynamically, after creating the component, so I now adding it here.
I'll think about checking the state later.